### DISCUSSION TITLE
Rule attribute shows only last message

### DISCUSSION THREAD

**Discussion by valpuia** [livewire/livewire#6734] [7. Bugs (archived)]
### Livewire version

v3.0.2

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

```php
#[Rule('required', message: 'Please provide a post title')]
#[Rule('min:3', message: 'This title is too short')]
public $title;
```
When apply like above validation in Rule attribute, the error always showing the last message only
![Screenshot from 2023-09-09 12-12-15](https://github.com/livewire/livewire/assets/28250303/c68dc56c-6d2e-47a7-a68f-6909ea159214)

But when using like this `#[Rule('required|min:3')] ` it's correct

**Expected**
Validation message should show according to it validate, if required then show required message

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by g-edgley** [comment]
Having the exact same issue.

![image](https://github.com/livewire/livewire/assets/87308231/11e724e5-e47a-4983-9de2-e68e8dea061a)

The required rule and message works fine. Once this is passed though, the 'alpha_dash' rule works, but returns the 'require' message should it fail, not the 'alpha_dash' message. Seems that only the first custom message is return of any validation failure, no matter the number of rules.

**Comment by PhiloNL** [comment]
Thanks for reporting! I've added a failing test https://github.com/livewire/livewire/pull/6761.

**Comment by PhiloNL** [comment]
@valpuia this should be fixed in the next release. Thanks for reporting: https://github.com/livewire/livewire/pull/6761
